packages/apps/plugins/plugin-table/src/components/ObjectTable/ObjectTable.tsx

```diff
@@ -171,6 +170,6 @@ const ObjectTableImpl: FC<ObjectTableProps> = ({ table, role, stickyHeader }) =>
           <pre className='flex-1'>{JSON.stringify((table.schema as any)?._schema, undefined, 2)}</pre>
         </div>
       )}
-    </DensityProvider>
+    </>
```

Comment:
Probably a good call. We should audit usages in devtools, stack, standalone, storybooks, etc. to check it's working as intended.